### ISSUE TITLE
There are no packages available for installation

### ISSUE THREAD

**Issue by davidsneighbour** [wbond/package_control#1351]
I am unable to install any packages because package control seems to be unable to download the list of available packages. Interesting in the debugging is that it says `Connection went away while trying to download https://packagecontrol.io`. The first time takes some seconds, the second and third time it comes quite fast after. 

The system is Lubuntu with LXDE as window manager. In Ubuntu with Gnome it worked - same setup, just different OS. 

```
Package Control: Fetching list of available packages and dependencies
  Platform: linux-x64
  Sublime Text Version: 3143
  Package Control Version: 3.3.0
Package Control: Download Debug
  URL: https://packagecontrol.io/channel_v3.json
  Timeout: 30
  Resolved IP: 50.116.34.243
  Resolved IPv6: 2600:3c02:e000:42::1
Package Control: Urllib Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Found system CA bundle at /usr/lib/ssl/certs/ca-certificates.crt (233394 bytes)
Package Control: Urllib HTTPS Debug General
  Connecting to packagecontrol.io on port 443
Package Control: Urllib HTTPS Debug General
  Upgrading connection to SSL using CA certs file at /home/patrick/.config/sublime-text-3/Packages/User/Package Control.merged-ca-bundle
  Using hostname "packagecontrol.io" for TLS SNI extension
  Successfully upgraded connection to packagecontrol.io:443 with SSL Using TLSv1/SSLv3 with cipher ECDHE-RSA-AES128-SHA
  Server SSL certificate:
    subject: CN=packagecontrol.io
    subject alt name: packagecontrol.io, www.packagecontrol.io
    expire date: Nov 27 23:59:59 2019 GMT
  Certificate validated for packagecontrol.io
Package Control: Urllib HTTPS Debug Write
  GET /channel_v3.json HTTP/1.1
  Accept-Encoding: bzip2,gzip,deflate
  User-Agent: Package Control v3.3.0
  Connection: Keep-Alive
  Host: packagecontrol.io
Package Control: Connection went away while trying to download https://packagecontrol.io/channel_v3.json, trying again
Package Control: Urllib Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Found system CA bundle at /usr/lib/ssl/certs/ca-certificates.crt (233394 bytes)
Package Control: Urllib HTTPS Debug General
  Connecting to packagecontrol.io on port 443
Package Control: Urllib HTTPS Debug General
  Upgrading connection to SSL using CA certs file at /home/patrick/.config/sublime-text-3/Packages/User/Package Control.merged-ca-bundle
  Using hostname "packagecontrol.io" for TLS SNI extension
  Successfully upgraded connection to packagecontrol.io:443 with SSL Using TLSv1/SSLv3 with cipher ECDHE-RSA-AES128-SHA
  Server SSL certificate:
    subject: CN=packagecontrol.io
    subject alt name: packagecontrol.io, www.packagecontrol.io
    expire date: Nov 27 23:59:59 2019 GMT
  Certificate validated for packagecontrol.io
Package Control: Urllib HTTPS Debug Write
  GET /channel_v3.json HTTP/1.1
  Accept-Encoding: bzip2,gzip,deflate
  User-Agent: Package Control v3.3.0
  Connection: Keep-Alive
  Host: packagecontrol.io
Package Control: Connection went away while trying to download https://packagecontrol.io/channel_v3.json, trying again
Package Control: Urllib Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Found system CA bundle at /usr/lib/ssl/certs/ca-certificates.crt (233394 bytes)
Package Control: Urllib HTTPS Debug General
  Connecting to packagecontrol.io on port 443
Package Control: Urllib HTTPS Debug General
  Upgrading connection to SSL using CA certs file at /home/patrick/.config/sublime-text-3/Packages/User/Package Control.merged-ca-bundle
  Using hostname "packagecontrol.io" for TLS SNI extension
  Successfully upgraded connection to packagecontrol.io:443 with SSL Using TLSv1/SSLv3 with cipher ECDHE-RSA-AES128-SHA
  Server SSL certificate:
    subject: CN=packagecontrol.io
    subject alt name: packagecontrol.io, www.packagecontrol.io
    expire date: Nov 27 23:59:59 2019 GMT
  Certificate validated for packagecontrol.io
Package Control: Urllib HTTPS Debug Write
  GET /channel_v3.json HTTP/1.1
  Accept-Encoding: bzip2,gzip,deflate
  User-Agent: Package Control v3.3.0
  Connection: Keep-Alive
  Host: packagecontrol.io
Package Control: Connection went away while trying to download https://packagecontrol.io/channel_v3.json, trying again
Package Control: Urllib Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Found system CA bundle at /usr/lib/ssl/certs/ca-certificates.crt (233394 bytes)
Package Control: Unable to download https://packagecontrol.io/channel_v3.json after 3 attempts
```

**Comment by nielsaka** [comment]
Same problem over here. On Windows there is no problem, but on Linux Mint (based on Ubuntu 16.04) I get the same error messages. I have tried the various instructions under [troubleshooting](https://packagecontrol.io/docs/troubleshooting) but to no avail.

I will give it a try on a another linux mint machine later.

```
Package Control: Skipping automatic upgrade, last run at 2018-04-03 19:55:02, next run at 2018-04-03 20:55:02 or after
Package Control: Fetching list of available packages and dependencies
  Platform: linux-x64
  Sublime Text Version: 3143
  Package Control Version: 3.3.0
Package Control: Download Debug
  URL: https://packagecontrol.io/channel_v3.json
  Timeout: 30
  Resolved IP: 50.116.34.243
  Resolved IPv6: 2600:3c02:e000:42::1
Package Control: Urllib Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Found system CA bundle at /usr/lib/ssl/certs/ca-certificates.crt (233394 bytes)
Package Control: Urllib HTTPS Debug General
  Connecting to packagecontrol.io on port 443
Package Control: Urllib HTTPS Debug General
  Upgrading connection to SSL using CA certs file at /home/niels/.config/sublime-text-3/Packages/User/Package Control.merged-ca-bundle
  Using hostname "packagecontrol.io" for TLS SNI extension
  Successfully upgraded connection to packagecontrol.io:443 with SSL Using TLSv1/SSLv3 with cipher ECDHE-RSA-AES128-SHA
  Server SSL certificate:
    subject: CN=packagecontrol.io
    subject alt name: packagecontrol.io, www.packagecontrol.io
    expire date: Nov 27 23:59:59 2019 GMT
  Certificate validated for packagecontrol.io
Package Control: Urllib HTTPS Debug Write
  GET /channel_v3.json HTTP/1.1
  User-Agent: Package Control v3.3.0
  Host: packagecontrol.io
  Accept-Encoding: bzip2,gzip,deflate
  Connection: Keep-Alive
Package Control: Connection went away while trying to download https://packagecontrol.io/channel_v3.json, trying again
```

**Comment by cristoferfb** [comment]
same problem on fedora

**Comment by nielsaka** [comment]
I tried to reproduce the behaviour on another machine, but package control worked fine. I then booted Linux Mint 18.3 Cinnamon from a live USB stick, installed sublime text 

    sudo apt-get install sublime-text

which apparently fetches sublime-text from a Mint repository

    Get:1 http://packages.linuxmint.com sylvia/import amd64 sublime-text amd64 3143 [8,190 kB]

I installed package control and then tried to install a package. On one machine this procedure works perfectly, on the other it fails with the above error messages. I have no clue what the reason could be.

**Comment by avisiedo** [comment]
Same issues, but different messages registered into the log. Using version for macos in High Sierra 10.13.4.

I had to make the Package Control installation by hand. Into the command line I was getting the same error "Host is down", however, using the browser I had no issue to download and browse the URL to be downloaded.

```
Package Control: Fetching list of available packages and dependencies
  Platform: osx-x64
  Sublime Text Version: 3170
  Package Control Version: 3.3.0
Package Control: Download Debug
  URL: https://packagecontrol.io/channel_v3.json
  Timeout: 30
  Resolved IP: 50.116.34.243
  Resolved IPv6: ::ffff:50.116.34.243
Package Control: Urllib Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Found previously exported CA bundle at /Users/avisiedo/Library/Application Support/Sublime Text 3/Packages/User/oscrypto-ca-bundle.crt (277136 bytes)
Package Control: Urllib HTTPS Debug General
  Connecting to packagecontrol.io on port 443
Package Control: Error downloading channel. URL error [Errno 64] Host is down downloading https://packagecontrol.io/channel_v3.json.
```

**Comment by dwasss** [comment]
I got a similar error due to SSL cert issues. I can confirm following the steps in https://github.com/wbond/package_control/issues/1302#issuecomment-352076439 (basically deleting anything Package Control related and installing the prerelease version) I was able to finally get it to work and install packages.

**Comment by bretttolbert** [comment]
I'm also seeing this issue. Here is the debug I got after enabling "debug" in Package Control settings:

```
Package Control: Connection went away while trying to download https://packagecontrol.io/channel_v3.json, trying again
```
Full debug:
```DPI scale (detected): 1
startup, version: 3176 linux x64 channel: stable
executable: /opt/sublime_text/sublime_text
working dir: /
packages path: /home/btolbert/.config/sublime-text-3/Packages
state path: /home/btolbert/.config/sublime-text-3/Local
zip path: /opt/sublime_text/Packages
zip path: /home/btolbert/.config/sublime-text-3/Installed Packages
ignored_packages: ["Vintage"]
pre session restore time: 0.075777
startup time: 0.140887
first paint time: 0.149336
reloading plugin Default.arithmetic
reloading plugin Default.auto_indent_tag
reloading plugin Default.block
reloading plugin Default.colors
reloading plugin Default.comment
reloading plugin Default.convert_color_scheme
reloading plugin Default.convert_syntax
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.install_package_control
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.profile
reloading plugin Default.quick_panel
reloading plugin Default.rename
reloading plugin Default.run_syntax_tests
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.settings
reloading plugin Default.show_scope_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin Default.ui
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin ShellScript.ShellScript
reloading plugin 0_package_control_loader.00-package_control
reloading plugin Package Control.1_reloader
reloading plugin Package Control.2_bootstrap
reloading plugin Package Control.Package Control
plugins loaded
Package Control: Unable to download https://packagecontrol.io/channel_v3.json after 3 attempts
Package Control: Unable to download https://packagecontrol.io/channel_v3.json after 3 attempts
Package Control: No updated packages
Unable to open /home/btolbert/Sbox/helloworld/CMakeLists.txt
Package Control: Unable to download https://packagecontrol.io/channel_v3.json after 3 attempts
Package Control: Unable to download https://packagecontrol.io/channel_v3.json after 3 attempts
Package Control: Unable to download https://packagecontrol.io/channel_v3.json after 3 attempts
reloading settings Packages/User/Package Control.sublime-settings
Package Control: Settings change detected, clearing cache
Package Control: Unable to download https://packagecontrol.io/channel_v3.json after 3 attempts
reloading settings Packages/User/Package Control.sublime-settings
Package Control: Settings change detected, clearing cache
Package Control: Fetching list of available packages and dependencies
  Platform: linux-x64
  Sublime Text Version: 3176
  Package Control Version: 3.3.0
Package Control: Download Debug
  URL: https://packagecontrol.io/channel_v3.json
  Timeout: 60
  Resolved IP: 50.116.34.243
  Resolved IPv6: 2600:3c02:e000:42::1
Package Control: Urllib Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Found system CA bundle at /usr/lib/ssl/certs/ca-certificates.crt (207436 bytes)
Package Control: Urllib HTTPS Debug General
  Connecting to packagecontrol.io on port 443
Package Control: Urllib HTTPS Debug General
  Upgrading connection to SSL using CA certs file at /home/btolbert/.config/sublime-text-3/Packages/User/Package Control.merged-ca-bundle
  Using hostname "packagecontrol.io" for TLS SNI extension
  Successfully upgraded connection to packagecontrol.io:443 with SSL Using TLSv1/SSLv3 with cipher ECDHE-RSA-AES128-GCM-SHA256
  Server SSL certificate:
    subject: CN=packagecontrol.io
    subject alt name: packagecontrol.io, www.packagecontrol.io
    expire date: Nov 27 12:00:00 2019 GMT
  Certificate validated for packagecontrol.io
Package Control: Urllib HTTPS Debug Write
  GET /channel_v3.json HTTP/1.1
  If-None-Match: "5bb505ff-52962"
  Connection: Keep-Alive
  User-Agent: Package Control v3.3.0
  Accept-Encoding: bzip2,gzip,deflate
  If-Modified-Since: Wed, 03 Oct 2018 18:10:07 GMT
  Host: packagecontrol.io
Package Control: Urllib HTTPS Debug Read
  HTTP/1.1 200 OK
  Server: nginx
  Date: Thu, 04 Oct 2018 17:37:53 GMT
  Content-Type: application/json
  Content-Length: 341082
  Last-Modified: Thu, 04 Oct 2018 17:30:07 GMT
  Connection: keep-alive
  ETag: "5bb64e1f-5345a"
  Content-Encoding: bzip2
  Vary: Accept-Encoding
  Strict-Transport-Security: max-age=15552000
  Accept-Ranges: bytes
Package Control: Caching https://packagecontrol.io/channel_v3.json in /home/btolbert/.config/sublime-text-3/Packages/User/Package Control.cache/01524fae79697630d0454ba3fabd9414
Package Control: Connection went away while trying to download https://packagecontrol.io/channel_v3.json, trying again
Package Control: Urllib Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Found system CA bundle at /usr/lib/ssl/certs/ca-certificates.crt (207436 bytes)
Package Control: Urllib HTTPS Debug General
  Connecting to packagecontrol.io on port 443
Package Control: Urllib HTTPS Debug General
  Upgrading connection to SSL using CA certs file at /home/btolbert/.config/sublime-text-3/Packages/User/Package Control.merged-ca-bundle
  Using hostname "packagecontrol.io" for TLS SNI extension
  Successfully upgraded connection to packagecontrol.io:443 with SSL Using TLSv1/SSLv3 with cipher ECDHE-RSA-AES128-GCM-SHA256
  Server SSL certificate:
    subject: CN=packagecontrol.io
    subject alt name: packagecontrol.io, www.packagecontrol.io
    expire date: Nov 27 12:00:00 2019 GMT
  Certificate validated for packagecontrol.io
Package Control: Urllib HTTPS Debug Write
  GET /channel_v3.json HTTP/1.1
  If-None-Match: "5bb505ff-52962"
  Connection: Keep-Alive
  User-Agent: Package Control v3.3.0
  Accept-Encoding: bzip2,gzip,deflate
  If-Modified-Since: Wed, 03 Oct 2018 18:10:07 GMT
  Host: packagecontrol.io
Package Control: Urllib HTTPS Debug Read
  HTTP/1.1 200 OK
  Server: nginx
  Date: Thu, 04 Oct 2018 17:37:54 GMT
  Content-Type: application/json
  Content-Length: 341082
  Last-Modified: Thu, 04 Oct 2018 17:30:07 GMT
  Connection: keep-alive
  ETag: "5bb64e1f-5345a"
  Content-Encoding: bzip2
  Vary: Accept-Encoding
  Strict-Transport-Security: max-age=15552000
  Accept-Ranges: bytes
Package Control: Caching https://packagecontrol.io/channel_v3.json in /home/btolbert/.config/sublime-text-3/Packages/User/Package Control.cache/01524fae79697630d0454ba3fabd9414
Package Control: Connection went away while trying to download https://packagecontrol.io/channel_v3.json, trying again
Package Control: Urllib Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Found system CA bundle at /usr/lib/ssl/certs/ca-certificates.crt (207436 bytes)
Package Control: Urllib HTTPS Debug General
  Connecting to packagecontrol.io on port 443
Package Control: Urllib HTTPS Debug General
  Upgrading connection to SSL using CA certs file at /home/btolbert/.config/sublime-text-3/Packages/User/Package Control.merged-ca-bundle
  Using hostname "packagecontrol.io" for TLS SNI extension
  Successfully upgraded connection to packagecontrol.io:443 with SSL Using TLSv1/SSLv3 with cipher ECDHE-RSA-AES128-GCM-SHA256
  Server SSL certificate:
    subject: CN=packagecontrol.io
    subject alt name: packagecontrol.io, www.packagecontrol.io
    expire date: Nov 27 12:00:00 2019 GMT
  Certificate validated for packagecontrol.io
Package Control: Urllib HTTPS Debug Write
  GET /channel_v3.json HTTP/1.1
  If-None-Match: "5bb505ff-52962"
  Connection: Keep-Alive
  User-Agent: Package Control v3.3.0
  Accept-Encoding: bzip2,gzip,deflate
  If-Modified-Since: Wed, 03 Oct 2018 18:10:07 GMT
  Host: packagecontrol.io
Package Control: Urllib HTTPS Debug Read
  HTTP/1.1 200 OK
  Server: nginx
  Date: Thu, 04 Oct 2018 17:37:54 GMT
  Content-Type: application/json
  Content-Length: 341082
  Last-Modified: Thu, 04 Oct 2018 17:30:07 GMT
  Connection: keep-alive
  ETag: "5bb64e1f-5345a"
  Content-Encoding: bzip2
  Vary: Accept-Encoding
  Strict-Transport-Security: max-age=15552000
  Accept-Ranges: bytes
Package Control: Caching https://packagecontrol.io/channel_v3.json in /home/btolbert/.config/sublime-text-3/Packages/User/Package Control.cache/01524fae79697630d0454ba3fabd9414
Package Control: Connection went away while trying to download https://packagecontrol.io/channel_v3.json, trying again
Package Control: Urllib Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Found system CA bundle at /usr/lib/ssl/certs/ca-certificates.crt (207436 bytes)
Package Control: Unable to download https://packagecontrol.io/channel_v3.json after 3 attempts```

**Comment by bretttolbert** [comment]
I got it working by clearing the cache with this:
```
rm ~/.config/sublime-text-3/Packages/User/Package\ Control.cache/*
```
